.github/pull_request_template.md

```diff
@@ -1,62 +1,9 @@
 <!-- Specify the issue ID that this pull request is solving -->
-Fix #XXX
+Fixes #XXX
```

Comment:
This could prevent issues from being automatically closed by a single PR, as we often close our issues through multiple PRs.
```suggestion
Refers to #XXX
```